t/full-stack.t

```diff
@@ -99,7 +99,7 @@ $driver->click_element_ok('dont-notify', 'id', 'Selected to not notify about tou
 $driver->click_element_ok('confirm',     'id', 'Clicked confirm about no tour');
 
 my $job_setup = $OpenQA::Test::FullstackUtils::JOB_SETUP;
-schedule_one_job_over_api_and_verify($driver, $job_setup . ' PAUSE_AT=shutdown');
+schedule_one_job_over_api_and_verify($driver, "PAUSE_AT=shutdown $job_setup");
```

Comment:
Why do you need to move this? I consider it more natural to have the "common parameters" first

Reply:
Because Mojolicious gives a higher precedence to the first parameter if there are multiple with the same name. So if you want to override you have to it this way. It's a minor security feature to help with injection attacks.

Reply:
Oh, and i had to change it because `openqa-cli api` now knows how to send multiple parameters with the same name.

Reply:
I guess there's quite a bit of potential for future confusion, so i'll replace the `$JOB_SETUP` variable with a function that generates proper parameters.